packages/engine/stdlib/package.json

```diff
@@ -37,7 +37,7 @@
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-promise": "4.3.1",
     "jest": "26.6.3",
-    "ts-jest": "26.5.3",
+    "ts-jest": "28.0.5",
```

Comment:
@Alfred-Mountfield Do you know for what we are using this?

Reply:
I do not. I reckon if the tests pass, then it's good enough 😅

Reply:
@Alfred-Mountfield, @TimDiekmann I upgraded Jest in https://github.com/hashintel/hash/pull/760, so this diff will be cancelled out when the other PR is merged. Turns out we're not running Jest in CI for `stdlib` 😅